deps/chakrashim/src/v8v8.cc

```diff
@@ -106,11 +106,18 @@ void V8::SetFlagsFromCommandLine(int *argc, char **argv, bool remove_flags) {
       if (remove_flags) {
         argv[i] = nullptr;
       }
+    } else if (equals("--trace-debug-json", arg) ||
+      equals("--trace_debug_json", arg)) {
+      g_trace_debug_json = true;
+      if (remove_flags) {
+        argv[i] = nullptr;
+      }
     } else if (remove_flags &&
                (startsWith(
                  arg, "--debug")  // Ignore some flags to reduce unit test noise
                 || startsWith(arg, "--harmony")
-                || startsWith(arg, "--stack-size="))) {
+                || startsWith(arg, "--stack-size=")
+                || startsWith(arg, "--nolazy"))) {
```

Comment:
Its like disabling deferred parsing in V8, VSCode by defaults add this flag so we need to ignore it.